### ISSUE TITLE
null exception using CombineWorker in another Combine in v0.5

### ISSUE THREAD

**Issue by ghost** [Maksimka101/combine#11]
after upgrading to version 0.5 and flutter 3.7 exact same code that previously worked doesnt work.
I use parts of my code(for parsing json etc) in different isolates, one of these isolates is a Combine isolate and in this case whenever CombineWorker() is called whether for initialization or execution it throws null exception.

```
[ERROR:flutter/runtime/dart_isolate.cc(1098)] Unhandled exception:
Null check operator used on a null value
#0      NativeIsolateFactory.create
native_isolate_factory.dart:30
#1      Combine.spawn
combine_singleton.dart:78
#2      CombineTaskExecutor.createExecutor
combine_task_executor.dart:38
#3      NativeWorkerManager.initialize
native_worker_manager.dart:35
#4      CombineWorkerImpl.initialize
combine_worker_impl.dart:41
```

<details>
<summary>flutter doctor</summary>

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.7.0, on Microsoft Windows [Version 10.0.22000.1516], locale en-US)
[✓] Windows Version (Installed version of Windows is version 10 or higher)
Checking Android licenses is taking an unexpectedly long time...[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
[✓] Chrome - develop for the web
[✓] Visual Studio - develop for Windows (Visual Studio Build Tools 2022 17.4.4)
[✓] Android Studio (version 2022.1)
[✓] VS Code (version 1.75.0)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

• No issues found!
```
</details>

note that it works as expected outside of another combine isolate(in main isolate and in isolates used by flutter_background_service that I use in other parts)
really appreciate your work

**Comment by Maksimka101** [comment]
Hello! Thanks for finding this issue!
Fix on it's way: #12

**Comment by Maksimka101** [comment]
New version with fix is published
